public/API/API_GetLeaderboardSubmissions.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ *  API_GetLeaderboardSubmissions - returns submissions for a Leaderboard for the given ID
+ *    i : leaderboardID
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
+ *  int         Count                       number of user entries returned in the response
+ *  int         Total                       number of user entries the leaderboard actually has overall
+ *  array       Results
+ *   object      [value]
+ *    int        Rank                       user's leaderboard rank
+ *    string     User                       name of user
+ *    string     Score                      string value of the proper ValueFormat of the leaderboard entry //this feels wrong
```

Comment:
Returning the raw score would probably be even more confusing, for example on speedrun leaderboards (check game ID 1).

Reply:
This part is tricky. I'm not sure what the proper thing to return is here. Do you have thoughts on it?

Reply:
Perhaps we should rename "Score" to "FormattedScore" and keep letting ValueFormat do the work.

It can be on the consumer to figure out what to do with the formatted score based on the format returned from another endpoint about the given leaderboard ID.

Reply:
Yeah, that's good with me.